src/material/datepicker/_datepicker-legacy-compat.scss

```diff
@@ -0,0 +1,85 @@
+@use '../button/button-theme';
+@use '../button/icon-button-theme';
+
+@mixin legacy-button-compat-theme($theme) {
```

Comment:
typo, "compact" :)

Reply:
Actually this is meant as "compat", an abbreviation for "compatibility", but if you think its confusing I can update it to the full word in a followup PR